tests/unit/test_pool.py

```diff
@@ -655,9 +832,45 @@ def test_ping_oldest_stale_and_not_exists(self):
 
         self.assertTrue(SESSIONS[0]._pinged)
         SESSIONS[1].create.assert_called()
+        self.assertNoSpans()
 
+    def test_spans_get_empty_pool(self):
+        pool = self._make_one()
+        database = _Database("name")
+        session1 = _Session(database)
+        database._sessions.append(session1)
+        try:
+            pool.bind(database)
+        except Exception:
+            pass
+
+        with trace_call("pool.Get", session1):
+            session = pool.get()
+            self.assertIsInstance(session, _Session)
+            self.assertIs(session._database, database)
+            # session.create.assert_called()
+            self.assertTrue(pool._sessions.empty())
+
+        self.assertSpanAttributes(
+            "pool.Get",
+            attributes=TestPingingPool.BASE_ATTRIBUTES,
+        )
+        wantEventNames = [
+            "Waiting for a session to become available",
+            "Acquired session",
+        ]
+        self.assertSpanEvents("pool.Get", wantEventNames)
+
+
+class TestTransactionPingingPool(OpenTelemetryBase):
```

Comment:
you can remove tests from TransactionPingingPool since it is deprecated

Reply:
Done, thanks!